src/esp/physics/bullet/BulletRigidObject.cpp

```diff
@@ -126,36 +156,10 @@ bool BulletRigidObject::initializeObject(
   //! Iterate through all mesh components for one object
   //! The components are combined into a convex compound shape
   bObjectShape_ = std::make_unique<btCompoundShape>();
-  for (const assets::CollisionMeshData& meshData : meshGroup) {
-    Corrade::Containers::ArrayView<Magnum::Vector3> v_data = meshData.positions;
-    Corrade::Containers::ArrayView<Magnum::UnsignedInt> ui_data =
-        meshData.indices;
+  Magnum::Matrix4 T;  // TODO: this should translate to the COM
```

Comment:
Doesn't look like this is used elsewhere, so deleted and do
```
constructBulletConvexCompoundFromMeshes(bObjectShape_, Magnum::Matrix4{Magnum::Math::IdentityInit}, meshGroup,
metaData.root);
```

Looks like Magnum does the right thing, (thank you @mosra) and `Magnum::Matrix4{}` would suffice :)